alfasim_sdk/hook_specs.py

```diff
@@ -1369,5 +1445,7 @@ def calculate_entrained_liquid_fraction(
         friction_factor,
         env_temperature,
         calculate_entrained_liquid_fraction,
+        # Internal Deposition Layer
```

Comment:
Remove the comment, it doesn't seem to add any useful information here

Reply:
- [ ] Remove the comment